src/pages/crm-reviews-page/CrmReviewsPage.tsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useState } from 'react'
+import axiosInstance from '../../axios/settings'
+
+import { ReviewsList } from './ReviewsList/ReviewsList'
+import PaginationCRM from '../../components/crm-pagination/PaginationCRM'
+import Box from '@mui/material/Box'
+import { Typography } from '@mui/material'
+
+import { Review } from '../../redux/feedbacks/slice'
+
+const PAGE_SIZE = 5
+
+const CrmReviewsPage = () => {
+  const [reviews, setReviews] = useState<Review[]>([])
+  const [page, setPage] = useState(0)
+
+  const fetchReviews = async () => {
+    try {
+      const offset = page * PAGE_SIZE
+      const { data } = await axiosInstance.get(
+        `/api/reviews/all_for_crm?limit=${PAGE_SIZE}&offset=${offset}`
+      )
+      setReviews(data)
+    } catch (err: unknown) {
+      if (err instanceof Error) {
+        console.error(err.message)
+      }
+    }
+  }
+
+  useEffect(() => {
+    fetchReviews()
+  }, [page])
+
+  const handleStatusChange = () => {
+    fetchReviews()
+  }
```

Comment:
same here, it is an unnecessary wrapper for the function

Reply:
in ReviewListItem, I need to update all reviews when the status changes, so I pass the function like this. Is there any other way to do it?

Reply:
```
<Component onUpdate={() => fetchReviews()} />
```
or
```
<Component onUpdate={fetchReviews)} />
```